demos/storybook/stories/selection-toolbar/with-full-config.stories.ts

```diff
@@ -0,0 +1,36 @@
+import { text } from '@storybook/addon-knobs';
+
+export const withFullConfig = (): any => ({
+    styles: [
+        `
+       ::ng-deep .pxb-selection-toolbar {
```

Comment:
Refactor this into a decorator so these styles apply to each story.
See InfoListItem for example on how to do this.

Also change the fixed width from 600px to use a percentage instead so it'll work on smaller screens:
![image](https://user-images.githubusercontent.com/6538289/88572965-07f77980-d00e-11ea-8416-b5bddfe3e674.png)